cms/templates/js/timed-examination-preference-editor.underscore

```diff
@@ -0,0 +1,39 @@
+<form>
+    <h3 class="modal-section-title"><%= gettext('Timed Exam') %></h3>
+    <div class="modal-section-content has-actions">
+        <div class='exam-time-list-fields'>
+            <ul class="list-fields list-input">
+                <li class="field field-checkbox checkbox-cosmetic">
+                    <input type="checkbox" id="id_timed_examination" name="timed_examination" class="input input-checkbox" />
+                    <label for="id_timed_examination" class="label">
+                      <i class="icon fa fa-check-square-o input-checkbox-checked"></i>
+                      <i class="icon fa fa-square-o input-checkbox-unchecked"></i>
+                      <%= gettext('This exam is timed') %>
```

Comment:
FYI: we always use ``<%-`` around ``gettext`` just in case the translations have characters that need to be escaped.